packages/yoastseo/src/languageProcessing/researches/keywordCountInUrl.js

```diff
@@ -103,7 +12,20 @@ function dehyphenateKeyphraseForms( topicForms ) {
  * @returns {{keyphraseLength: int, percentWordMatches: int}} The length of the keyphrase and the percentage of keyphrase forms that has been found.
  */
 function keywordCountInSlug( paper, researcher ) {
-	const topicForms = dehyphenateKeyphraseForms( researcher.getResearch( "morphology" ) );
```

Comment:
We can get rid of this step since `getWordForms` will already split the keyphrase on hyphens.